.sample_env

```diff
@@ -0,0 +1,18 @@
+DEBUG=True
+TEMPLATE_DEBUG=True
+SQL_DEBUG=True
+CACHE_BACKEND=
+ENABLE_SENTRY=False
+SENTRY_REDIS=False
+SENTRY_CELERY=False
+SENTRY_DSN=
+ALLOWED_HOSTS=localhost
+POD_IP=127.0.0.1
```

Comment:
@mirzakhany Is it needed to have it here?

Reply:
I listed all the environment variables we use in settings
which one do you think is irrelevant?

Reply:
We used it in `src/proj/settings/base.py` line `57`
@mirzakhany 
```python
ALLOWED_HOSTS.append(os.environ.get("POD_IP", ""))
```